src/main/java/io/fusionauth/load/FusionAuthBaseWorker.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2012-2025, FusionAuth, All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package io.fusionauth.load;
+
+import java.util.UUID;
+
+import io.fusionauth.client.FusionAuthClient;
+
+/**
+ * Worker to test creating EmailVerification Ids for users.
```

Comment:
This comment is wrong. Possible suggestion.
```suggestion
 * Base worker to create deterministic Ids for FusionAuth load tests.
```